src/nms-openwire.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Apache.NMS.ActiveMQ</RootNamespace>
+    <AssemblyName>Apache.NMS.ActiveMQ</AssemblyName>
+    <Version>1.8.0</Version>
+    <Company>Apache Software Foundation</Company>
+    <Product>Apache ActiveMQ</Product>
```

Comment:
This should be Author tag.

Product Tag should be: Apache NMS OpenWire

Reply:
Should be fixed.